src/server/session.ts

```diff
@@ -711,7 +711,9 @@ namespace ts.server {
                     catch { } // tslint:disable-line no-empty
                 }
 
-                if (err.message && err.message.indexOf(`Could not find sourceFile:`) !== -1) {
+
+                if (err.ProgramFiles) {
+                    msg += `\n\nProgram files: {JSON.stringify(error.ProgramFiles}}\n`;

```

Comment:
This needs to be 
 msg += `\n\nProgram files: ${JSON.stringify(error.ProgramFiles)}\n`;

Reply:
That's what I get for inspecting the property in the debugger and not actually reading the log...

Reply:
#32789